spec/requests/classified_listings_spec.rb

```diff
@@ -272,5 +326,140 @@
         expect(org_listing.reload.bumped_at >= previous_bumped_at).to eq(true)
       end
     end
+
+    context "when the publish action is called" do
```

Comment:
It seems like the redirect logic on updating is not tested, or am I missing something?